src/organisationClient.ts

```diff
@@ -43,12 +43,13 @@ export class OrganisationClient {
             .trim()
             .toLowerCase()
             .includes(name);
-        })
+        }).sort((organisationOne, organisationsTwo) =>
+            organisationOne.name.localeCompare(organisationsTwo.name))
```

Comment:
why not:

```
.sort(alphabeticalOrder)
```

and then add a private:
```
private alphabeticalOrder(organisationOne, organisationTwo): Number {
    return organisationOne.name.localeCompare(organisationsTwo.name);
}
```

I didn't know what `localeCompare` does :) I suppose majority of devs wouldn't know.

Reply:
Very good point! Ill update now

Reply:
Updated, but in a different manor. Take a look and see what you think @yarpo 


@adamjhc